config/sync/block_content.type.footer.yml

```diff
@@ -3,6 +3,6 @@ langcode: da
 status: true
 dependencies: {  }
 id: footer
-label: Footer
+label: 'Footer (1)'
```

Comment:
Change this to not include columns as this is not available in the layout builder and can only be added in content block section.